ncolony/directory_monitor.py

```diff
@@ -24,7 +25,7 @@ def checker(location, receiver):
     path = filepath.FilePath(location)
     files = set()
     filesContents = {}
-    def _check():
+    def _check(path):
```

Comment:
Why is this better than a closure?

Reply:
It is more amenable to testing. I can check that the path argument is properly passed in. This is probably not a great test, so I opened #49